[2023-11-23T11:40:25.532+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Projet1.Extract manual__2023-11-23T11:40:21.582145+00:00 [queued]>
[2023-11-23T11:40:25.545+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Projet1.Extract manual__2023-11-23T11:40:21.582145+00:00 [queued]>
[2023-11-23T11:40:25.546+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-23T11:40:25.562+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Extract> on 2023-11-23 11:40:21.582145+00:00
[2023-11-23T11:40:25.567+0000] {standard_task_runner.py:57} INFO - Started process 2342 to run task
[2023-11-23T11:40:25.570+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Projet1', 'Extract', 'manual__2023-11-23T11:40:21.582145+00:00', '--job-id', '558', '--raw', '--subdir', 'DAGS_FOLDER/Projet.py', '--cfg-path', '/tmp/tmphupowfve']
[2023-11-23T11:40:25.573+0000] {standard_task_runner.py:85} INFO - Job 558: Subtask Extract
[2023-11-23T11:40:25.622+0000] {task_command.py:416} INFO - Running <TaskInstance: Projet1.Extract manual__2023-11-23T11:40:21.582145+00:00 [running]> on host b36ebcee5948
[2023-11-23T11:40:25.705+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Projet1' AIRFLOW_CTX_TASK_ID='Extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-23T11:40:21.582145+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-23T11:40:21.582145+00:00'
[2023-11-23T11:40:25.935+0000] {warnings.py:109} WARNING - /opt/***/dags/Projet.py:17: DtypeWarning: Columns (0) have mixed types. Specify dtype option on import or set low_memory=False.
  df_urgences = pd.read_csv(csv_file_path_urgences,  delimiter=';' )

[2023-11-23T11:40:26.414+0000] {logging_mixin.py:151} INFO - 101
[2023-11-23T11:40:26.417+0000] {logging_mixin.py:151} INFO - 99
[2023-11-23T11:40:26.428+0000] {logging_mixin.py:151} INFO -    Code tranches d'age             Age
0                    0       tous Ã¢ges
1                    1         0-4 ans
2                    2        5-14 ans
3                    3       15-44 ans
4                    4       45-64 ans
5                    5       65-74 ans
6                    6  75 ans ou plus
[2023-11-23T11:40:26.429+0000] {logging_mixin.py:151} INFO - Index(['Code tranches d'age', 'Age'], dtype='object')
[2023-11-23T11:40:26.430+0000] {logging_mixin.py:151} INFO - Index(['code_age', 'agess'], dtype='object')
[2023-11-23T11:40:26.430+0000] {logging_mixin.py:151} INFO - zmmmmmmarr 
[2023-11-23T11:40:26.431+0000] {logging_mixin.py:151} INFO - Index(['dep', 'date_de_passage', 'sursaud_cl_age_corona', 'nbre_pass_corona',
       'nbre_pass_tot', 'nbre_hospit_corona', 'nbre_pass_corona_h',
       'nbre_pass_corona_f', 'nbre_pass_tot_h', 'nbre_pass_tot_f',
       'nbre_hospit_corona_h', 'nbre_hospit_corona_f', 'nbre_acte_corona',
       'nbre_acte_tot', 'nbre_acte_corona_h', 'nbre_acte_corona_f',
       'nbre_acte_tot_h', 'nbre_acte_tot_f'],
      dtype='object')
[2023-11-23T11:40:26.448+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T11:40:26.516+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T11:40:26.552+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T11:40:33.063+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1890, in _execute_context
    self.dialect.do_executemany(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 982, in do_executemany
    context._psycopg2_fetched_rows = xtras.execute_values(
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
psycopg2.errors.ForeignKeyViolation: insert or update on table "Urgences" violates foreign key constraint "urgences_dep_fkey"
DETAIL:  Key (dep)=(0) is not present in table "Departements".


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Projet.py", line 118, in extract_data
    df_urgences.to_sql('Urgences', postgres_sql_upload.get_sqlalchemy_engine(), if_exists='append', chunksize=1000, index=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/generic.py", line 2878, in to_sql
    return sql.to_sql(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 769, in to_sql
    return pandas_sql.to_sql(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1920, in to_sql
    total_inserted = sql_engine.insert_records(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1470, in insert_records
    raise err
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1461, in insert_records
    return table.insert(chunksize=chunksize, method=method)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1023, in insert
    num_inserted = exec_insert(conn, keys, chunk_iter)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 929, in _execute_insert
    result = conn.execute(self.table.insert(), data)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1385, in execute
    return meth(self, multiparams, params, _EMPTY_EXECUTION_OPTS)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 334, in _execute_on_connection
    return connection._execute_clauseelement(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1577, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1953, in _execute_context
    self._handle_dbapi_exception(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 2134, in _handle_dbapi_exception
    util.raise_(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1890, in _execute_context
    self.dialect.do_executemany(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 982, in do_executemany
    context._psycopg2_fetched_rows = xtras.execute_values(
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
sqlalchemy.exc.IntegrityError: (psycopg2.errors.ForeignKeyViolation) insert or update on table "Urgences" violates foreign key constraint "urgences_dep_fkey"
DETAIL:  Key (dep)=(0) is not present in table "Departements".

[SQL: INSERT INTO "Urgences" (dep, date_de_passage, sursaud_cl_age_corona, nbre_pass_corona, nbre_pass_tot, nbre_hospit_corona, nbre_pass_corona_h, nbre_pass_corona_f, nbre_pass_tot_h, nbre_pass_tot_f, nbre_hospit_corona_h, nbre_hospit_corona_f, nbre_acte_corona, nbre_acte_tot, nbre_acte_corona_h, nbre_acte_corona_f, nbre_acte_tot_h, nbre_acte_tot_f) VALUES (%(dep)s, %(date_de_passage)s, %(sursaud_cl_age_corona)s, %(nbre_pass_corona)s, %(nbre_pass_tot)s, %(nbre_hospit_corona)s, %(nbre_pass_corona_h)s, %(nbre_pass_corona_f)s, %(nbre_pass_tot_h)s, %(nbre_pass_tot_f)s, %(nbre_hospit_corona_h)s, %(nbre_hospit_corona_f)s, %(nbre_acte_corona)s, %(nbre_acte_tot)s, %(nbre_acte_corona_h)s, %(nbre_acte_corona_f)s, %(nbre_acte_tot_h)s, %(nbre_acte_tot_f)s)]
[parameters: ({'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 30, 0, 0), 'sursaud_cl_age_corona': 4, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 114.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 4.0, 'nbre_acte_tot': 30.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 30, 0, 0), 'sursaud_cl_age_corona': 5, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 86.0, 'nbre_hospit_corona': 1.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 14.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 30, 0, 0), 'sursaud_cl_age_corona': 6, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 132.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 14.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 0, 'nbre_pass_corona': 3.0, 'nbre_pass_tot': 673.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 2.0, 'nbre_pass_corona_f': 1.0, 'nbre_pass_tot_h': 345.0, 'nbre_pass_tot_f': 328.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 9.0, 'nbre_acte_tot': 159.0, 'nbre_acte_corona_h': 4.0, 'nbre_acte_corona_f': 5.0, 'nbre_acte_tot_h': 67.0, 'nbre_acte_tot_f': 92.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 1, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 35.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 18.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 2, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 53.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 19.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 3, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 212.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 6.0, 'nbre_acte_tot': 70.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 4, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 139.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 3.0, 'nbre_acte_tot': 28.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}  ... displaying 10 of 1000 total bound parameter sets ...  {'dep': 0.0, 'date_de_passage': datetime.datetime(2023, 3, 23, 0, 0), 'sursaud_cl_age_corona': 1, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 8.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 43.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 0.0, 'date_de_passage': datetime.datetime(2023, 3, 23, 0, 0), 'sursaud_cl_age_corona': 2, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 14.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 1.0, 'nbre_acte_tot': 57.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0})]
(Background on this error at: https://sqlalche.me/e/14/gkpj)
[2023-11-23T11:40:33.117+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=Projet1, task_id=Extract, execution_date=20231123T114021, start_date=20231123T114025, end_date=20231123T114033
[2023-11-23T11:40:33.135+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 558 for task Extract ((psycopg2.errors.ForeignKeyViolation) insert or update on table "Urgences" violates foreign key constraint "urgences_dep_fkey"
DETAIL:  Key (dep)=(0) is not present in table "Departements".

[SQL: INSERT INTO "Urgences" (dep, date_de_passage, sursaud_cl_age_corona, nbre_pass_corona, nbre_pass_tot, nbre_hospit_corona, nbre_pass_corona_h, nbre_pass_corona_f, nbre_pass_tot_h, nbre_pass_tot_f, nbre_hospit_corona_h, nbre_hospit_corona_f, nbre_acte_corona, nbre_acte_tot, nbre_acte_corona_h, nbre_acte_corona_f, nbre_acte_tot_h, nbre_acte_tot_f) VALUES (%(dep)s, %(date_de_passage)s, %(sursaud_cl_age_corona)s, %(nbre_pass_corona)s, %(nbre_pass_tot)s, %(nbre_hospit_corona)s, %(nbre_pass_corona_h)s, %(nbre_pass_corona_f)s, %(nbre_pass_tot_h)s, %(nbre_pass_tot_f)s, %(nbre_hospit_corona_h)s, %(nbre_hospit_corona_f)s, %(nbre_acte_corona)s, %(nbre_acte_tot)s, %(nbre_acte_corona_h)s, %(nbre_acte_corona_f)s, %(nbre_acte_tot_h)s, %(nbre_acte_tot_f)s)]
[parameters: ({'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 30, 0, 0), 'sursaud_cl_age_corona': 4, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 114.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 4.0, 'nbre_acte_tot': 30.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 30, 0, 0), 'sursaud_cl_age_corona': 5, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 86.0, 'nbre_hospit_corona': 1.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 14.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 30, 0, 0), 'sursaud_cl_age_corona': 6, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 132.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 14.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 0, 'nbre_pass_corona': 3.0, 'nbre_pass_tot': 673.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 2.0, 'nbre_pass_corona_f': 1.0, 'nbre_pass_tot_h': 345.0, 'nbre_pass_tot_f': 328.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 9.0, 'nbre_acte_tot': 159.0, 'nbre_acte_corona_h': 4.0, 'nbre_acte_corona_f': 5.0, 'nbre_acte_tot_h': 67.0, 'nbre_acte_tot_f': 92.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 1, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 35.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 18.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 2, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 53.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 19.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 3, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 212.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 6.0, 'nbre_acte_tot': 70.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 29.0, 'date_de_passage': datetime.datetime(2023, 8, 31, 0, 0), 'sursaud_cl_age_corona': 4, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 139.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 3.0, 'nbre_acte_tot': 28.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}  ... displaying 10 of 1000 total bound parameter sets ...  {'dep': 0.0, 'date_de_passage': datetime.datetime(2023, 3, 23, 0, 0), 'sursaud_cl_age_corona': 1, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 8.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 0.0, 'nbre_acte_tot': 43.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0}, {'dep': 0.0, 'date_de_passage': datetime.datetime(2023, 3, 23, 0, 0), 'sursaud_cl_age_corona': 2, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 14.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': 0.0, 'nbre_pass_corona_f': 0.0, 'nbre_pass_tot_h': 0.0, 'nbre_pass_tot_f': 0.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 0.0, 'nbre_acte_corona': 1.0, 'nbre_acte_tot': 57.0, 'nbre_acte_corona_h': 0.0, 'nbre_acte_corona_f': 0.0, 'nbre_acte_tot_h': 0.0, 'nbre_acte_tot_f': 0.0})]
(Background on this error at: https://sqlalche.me/e/14/gkpj); 2342)
[2023-11-23T11:40:33.186+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-23T11:40:33.209+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
